### PR TITLE
Added Duality Docs Repo

### PR DESCRIPTION
**Pull request by ilexp**
Duality is a 2D game development framework that's [already part of upforgrabs](https://github.com/up-for-grabs/up-for-grabs.net/blob/gh-pages/_data/projects/duality.yml) for a few years. We recently split up our documentation, so it's now in a separate project, along with all issues related to the docs themselves. To get them listed again, I've added the docs project to upforgrabs separately, as we appreciate contributions to documentation as much as to the project core.

Let me know whether you think this makes sense for upforgrabs - not sure if tech writing should be a part of it, or whether the focus should be just coding itself, in which case just let me know and close the PR unmerged.

### CODE REVIEW

File: _data/projects/duality-docs.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Duality Docs
+desc: >
```
Comment (ritwik12) [inline _data/projects/duality-docs.yml]: @ilexp I think this is causing the problem in passing build for Travis CI, please refer other .yml in this repositories. You should have a single line description.

Comment (ilexp) [reply _data/projects/duality-docs.yml]: Oh, I didn't realize this was failing the build. Sorry! Will look into this and update the PR.

**Comment by ilexp** [comment]
Updated the description to be single-line, and while at it also did the same for the description of the already existing / merged duality.yml as well.

**Comment by ritwik12** [comment]
@ilexp Thanks!!

**Pull request merged**